src/context/fetcher/TenantBillingDetails.tsx

```diff
@@ -62,8 +62,21 @@ const TenantBillingDetailsContextProvider = ({
 // TODO (optimization): Consider defining a hook that returns an array of mapped tenant names.
 //   The majority of the components that call useTenantDetails do so to extract a memoized
 //   array of tenant names.
-const useTenantBillingDetails = () => {
+export const useTenantBillingDetails = () => {
     return useContext(TenantContext);
 };
 
-export { TenantBillingDetailsContextProvider, useTenantBillingDetails };
+export const useTenantUsesExternalPayment = (selectedTenant: string) => {
+    const { tenantBillingDetails } = useTenantBillingDetails();
+
+    return useMemo(() => {
+        return !tenantBillingDetails
+            ? false
+            : tenantBillingDetails.some((tenantBillingDetail) => {
+                  return (
+                      tenantBillingDetail.tenant === selectedTenant &&
+                      tenantBillingDetail.payment_provider === 'external'
+                  );
+              });
```

Comment:
Is there a benefit to using a ternary as opposed to the nullish coalescing operator?